spec/support/deploy_status_checker_helper.rb

```diff
@@ -0,0 +1,29 @@
+module DeployStatusCheckerHelper
```

Comment:
good idea to extract this! Should we also do the same in `deploy_status_checker_spec.rb`'s `#check` section?